.github/workflows/main.yml

```diff
@@ -16,7 +16,7 @@ jobs:
       uses: actions/checkout@v2
       with:
         fetch-depth: 0
-    - uses: conda-incubator/setup-miniconda@v2
+    - uses: conda-incubator/setup-miniconda@v2.1.1
       with:
         auto-update-conda: false
         miniconda-version: py37_4.10.3
```

Comment:
`setup-miniconda>=2.2` does not handle the `py37_` prefix, see https://github.com/conda-incubator/setup-miniconda/issues/261 .
(Over all, it would make sense to update versions to test against. That would need knowledge of what versions of `conda` Anaconda intends to officially/unofficially support; i.e., not something I know and something better done in a separate PR.)